docs/reference.cue

```diff
@@ -1 +1,36 @@
 package metadata
+
+#Fields: [Name=string]: #Fields | _
+
+remap: {
+  errors: [Name=string]: {
+    description: string
+    name: Name
+  }
+
+  functions: [Name=string]: {
+    arguments: [
+      ...{
+        required: bool,
+
+        if !required {
+          name: string
+        }
+
+        type: "float" | "int" | "string"
+      }
+    ]
+    category: "coerce" | "parse"
```

Comment:
Not sure which categories we still need to add, but there are a few that don't neatly fit these two:

* `contains`
* `split`
* `now`
* `sha1` (etc)
* `strip_whitespace`
* `truncate`
* `upcase`
* `uuid_v4`

_and a few more_